src/main/java/com/lookfirst/wepay/WePayApi.java

```diff
@@ -293,7 +293,7 @@ private HttpURLConnection getConnectionOnce(String uri, String postJson, String
 		if (postJson != null) {
 			conn.setDoOutput(true); // Triggers POST.
 		}
-		conn.setRequestProperty("Content-Type", "application/json; charset=utf-8");
+		conn.setRequestProperty("Content-Type", "application/json");
```

Comment:
WePay is wrong. They should fix their stuff if this totally valid header is causing an error. Looking at their API source code, that is wrong too... https://github.com/wepay/WePay-Java-SDK/blob/master/src/main/java/com/wepay/net/WePayResource.java#L48